tests/test_delete_plan_message_keys.py

```
@@ -0,0 +1,142 @@
+        pytest.skip("node not available to evaluate the frontend message helper")
+
+    src = _MESSAGES_JS.read_text(encoding="utf-8").replace("export function", "function")
+    script = tmp_path / "summarize_eval.mjs"
+    script.write_text(
+        src
+        + "\nconst __input = "
+        + json.dumps(messages)
+        + ";\nprocess.stdout.write(JSON.stringify(summarizeMessages(__input)));\n",
+        encoding="utf-8",
+    )
+    proc = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)